streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsRebalanceListener.java

```diff
@@ -52,8 +53,16 @@ public void onPartitionsAssigned(final Collection<TopicPartition> partitions) {
         // NB: all task management is already handled by:
         // org.apache.kafka.streams.processor.internals.StreamsPartitionAssignor.onAssignment
         if (assignmentErrorCode.get() == AssignorError.INCOMPLETE_SOURCE_TOPIC_METADATA.code()) {
-            log.error("Received error code {}", assignmentErrorCode.get());
+            log.error("Received error code {}", AssignorError.INCOMPLETE_SOURCE_TOPIC_METADATA);
             throw new MissingSourceTopicException("One or more source topics were missing during rebalance");
+        } else if (assignmentErrorCode.get() == AssignorError.VERSION_PROBING.code()) {
+            log.info("Received error code {}", AssignorError.VERSION_PROBING);
```

Comment:
Maybe we shouldn't say "error" when it's not an error. I'm just imagining the mailing list questions that will start pouring in...

```suggestion
            log.info("Received version probing code {}", AssignorError.VERSION_PROBING);
```